core-terms-vdp.md

```diff
@@ -0,0 +1,49 @@
+# [organization] Vulnerability Disclosure Policy
+
+## Introduction
+
+[organization] welcomes feedback from security researchers and the general public to help improve our security. If you believe you have discovered a vulnerability, privacy issue, exposed data, or other security issues in any of our assets, we want to hear from you. This policy outlines steps for reporting vulnerabilities to us, what we expect, what you can expect from us.
+
+## Scope
+
+This policy applies to any digital assets owned, operated, or maintained by [organization], including public facing websites.
+
```

Comment:
Looking at https://github.com/disclose/dioterms/pull/5/files#diff-d49350228ef1f6c3dc92a1a3ddf500d56a685ee18a0f9631fe5066d206711482R7-R12, it seems like the idea is that a "Bug Bounty Program" has to define custom in/out of scope assets and rewards, while a VDP should use this specific language for its "Scope" section?

I wasn't sure if this implied that a VDP always needed to be relevant to "any digital assets owned, operated, or maintained" by the org. I can see cases where you want a "scoped" VDP in some sense – e.g. if you sell an on-prem product and want to scope the program to instances of the on-prem product that the researcher has permission to test. There are other edge cases like third-party SaaS services purchased by the VDP owner where they might have some power to modify the configuration of their tenant, but may not be able to authorize certain types of testing against that web infrastructure as it's not actually owned by the VDP owner.

Reply:
That's correct. The intent here would be to more clearly reflect and reinforce that a VDP ultimately doesn't create control over any testing a researcher does (or, indeed, where a security issue may be identified by someone who isn't doing proactive security research). In contrast, a bounty program may have incentives in one area and not in others (cc: @cablej)